src/material.rs

```diff
@@ -182,7 +212,45 @@ impl Default for Pbr {
     }
 }
 
+#[doc(hidden)]
```

Comment:
`pub(crate)`?

Reply:
No, can't expose private types in public `enum Material`

Reply:
Hmm... I'm not happy about having two separate material kinds. Perhaps we should make `Material` opaque for now?